SwrveSDK/SDK/Track/Swrve.m

```diff
@@ -1915,6 +1939,16 @@ - (void) queueDeviceProperties
     [self userUpdate:self.deviceInfo];
 }
 
+- (BOOL) canSendExtraLogs {
+    return self->extraLogs;
+}
+
+- (void) extraLogEvent:(NSString*)name payload:(NSDictionary*)payload {
```

Comment:
Should this have a (reasonable) level of rate limiting if someone is doing something crazy with their integration? I'm not sure what the worst-case-scenario that's possible is right now with an integration / set of active campaigns for us here. We don't want an unbounded amount of debug logging to be sent, for all users, to our sinks if we flip a feature flag.